src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -494,6 +527,24 @@ public Task ExecuteCommand(PSCommand psCommand)
         }
 
         /// <summary>
+        /// Loads PowerShell profiles for the host from the
+        /// standard system locations.  Only the profile paths which
+        /// exist are loaded.
+        /// </summary>
+        /// <returns>A Task that can be awaited for completion.</returns>
+        public async Task LoadProfilesForHost()
```

Comment:
Not exactly sure how I feel about this method name.  I suppose I don't like it because `LoadProfiles` feels too vague and `LoadProfilesForHost` sounds like it's expecting a parameter.  I want to indicate that the function causes the profile paths in the `$profile` variable to be loaded.  Any thoughts?